build.sh

```diff
@@ -166,11 +166,35 @@ __rootbinpath="$__scriptpath/bin"
 __msbuildpackageid="Microsoft.Build.Mono.Debug"
 __msbuildpackageversion="14.1.0.0-prerelease"
 __msbuildpath=$__packageroot/$__msbuildpackageid.$__msbuildpackageversion/lib/MSBuild.exe
-__BuildArch=x64
 __buildmanaged=false
 __buildnative=false
 __TestNugetRuntimeId=win7-x64
 
+# Use uname to determine what the CPU is.
+CPUName=$(uname -p)
+case $CPUName in
+    i686)
```

Comment:
I decided to remove them since here I think we can say it is supported if it compiles (unless there are direct dependencies on CoreCLR in the native code)? What do you think?